apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -56,7 +56,8 @@ const DepartureList = ({
           </div>
           <a
             className="open-schedule"
-            href={`../schedules/${route.id}/line?schedule_direction[direction_id]=${directionId}&schedule_direction[variant]=${tripForSelectedRoutePattern?.route_pattern_id}&schedule_finder[direction_id]=${directionId}&schedule_finder[origin]=${stop.id}`}
+            href={`/schedules/${route.id}/line?schedule_direction[direction_id]=${directionId}&schedule_direction[variant]=${tripForSelectedRoutePattern?.route_pattern_id}&schedule_finder[direction_id]=${directionId}&schedule_finder[origin]=${stop.id}`}
+            data-turbolinks="false"
```

Comment:
This attribute opts out of involving [Turbolinks](https://github.com/turbolinks/turbolinks) in this page navigation. It causes minor issues sometimes.

Reply:
This might also fix the page schedule modal not loading on every other visit as well

Reply:
Ah, Github lets you view the whole file tree for the branch - can see the (correctly named) file here https://github.com/mbta/dotcom/blob/cbj/fix-link-to-schedule-finder/apps/site/assets/ts/models/predictions.d.ts

Looks like it was my bad for changing that a few days ago and forgetting to update those tests!